README.md

```diff
@@ -15,7 +15,7 @@ We recommend using [`yarn`](https://yarnpkg.com/en/). Run `yarn` to install depe
 
 Once the repository has been cloned, you can build the components with:
 
-`npx gulp build`
+`npm gulp build`
```

Comment:
Nope, should be `npx`.

Reply:
Oops. Didn't mean to commit that. Caught on after a bit.